examples/service_envoy/listeners.pb

```diff
@@ -0,0 +1,16 @@
+listeners {
```

Comment:
That's the proto syntax for a repeated (array) field. E.g. if you have:

```
message Blah {
  string value = 1;
}

repeated Blah foo = 1;
```

its textproto representation might be:

```
foo {
  value: "asdf"
}

foo {
  value: "dfsgfd"
}
```